### ISSUE TITLE
Failed to initialize client... ERROR: Error: Invalid response received from PTC login

### ISSUE THREAD

**Issue by rrbailey89** [cyraxx/pogobuf#97]
When using the ToS acceptor script I seem to receive this error 3 times and then it moves on to accepting the ToS for some of the accounts. I have been told this an issue with pogobuf so I am here submitting this issue.

```
Failed to initialize client... ERROR: Error: Invalid response received from PTC login
    at Error (native)
    at Request.self.request [as _callback] (F:\Ray\Downloads\PoGoMap\ptc-acc-gen\node_modules\pogobuf\pogobuf\pogobuf.ptclogin.js:108:28)
    at Request.self.callback (F:\Ray\Downloads\PoGoMap\ptc-acc-gen\node_modules\request\request.js:187:22)
    at emitTwo (events.js:106:13)
    at Request.emit (events.js:191:7)
    at Request.<anonymous> (F:\Ray\Downloads\PoGoMap\ptc-acc-gen\node_modules\request\request.js:1044:10)
    at emitOne (events.js:96:13)
    at Request.emit (events.js:188:7)
    at IncomingMessage.<anonymous> (F:\Ray\Downloads\PoGoMap\ptc-acc-gen\node_modules\request\request.js:965:12)
    at emitNone (events.js:91:20)
    at IncomingMessage.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:975:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

**Comment by MakuraYami** [comment]
At first glance this seems like incorrect login information. 

That said I'll assume you checked that. Could you provide some more information? Which package are you using, which code is causing this exactly. 

You seem to be using some scanner map thing, there could be all kind of things happening in between causing this. 

Also it could just be PTC servers temporarily unavailable.

**Comment by rrbailey89** [comment]
@MakuraYami My apologies I did not think to check and verify the login credentials. The repo that I was using to auto generate the accounts is here https://github.com/FrostTheFox/ptc-acc-gen the accounts are generated automatically in my case I opted to use randomly generated passwords too then they are passed to the pogoplayer repo https://github.com/MRokas/PogoPlayer which is a submodule within the acc-gen. The pogoplayer attempts to verify the ToS which is when I was receiving the above error. After manually attempting to login it was found that the pass was indeed incorrect on 3 of the accounts.
